### PR TITLE
[13.0][IMP] stock_reserve_sale: Show Reserve Stock button in sent orders.

### PR DESCRIPTION
**Pull request by victoralmau**
Show Reserve Stock button in sent orders.

Please @pedrobaeza and @CarlosRoca13 can you review it?

@Tecnativa TT39113

### CODE REVIEW

File: stock_reserve_sale/migrations/13.0.1.1.0/post-migration.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    orders = env["sale.order"].search(
+        [("state", "=", "sent"), ("is_stock_reservable", "=", False)]
+    )
+    orders._compute_stock_reservation()
```
Comment (pedrobaeza) [inline stock_reserve_sale/migrations/13.0.1.1.0/post-migration.py:12]: Why you need this?

Comment (pedrobaeza) [reply stock_reserve_sale/migrations/13.0.1.1.0/post-migration.py:12]: OK, seen.

**Review by pedrobaeza — APPROVED** [review]

**Comment by victoralmau** [comment]
Ping @CarlosRoca13

**Review by CarlosRoca13 — APPROVED** [review]

**Pull request merged**